src/main/java/eu/enexa/service/EnexaServiceImpl.java

```diff
@@ -39,6 +44,10 @@ public class EnexaServiceImpl implements EnexaService {
     private ModuleManager moduleManager;
 
     private static final String metaDataEndpoint = System.getenv("ENEXA_META_DATA_ENDPOINT");
+    private static final String appName = "app1";
```

Comment:
The application name should be sent by the application. If we keep it static and final like this, all applications will have the same name.

Reply:
I just define it in the application.properties, If different applications need to use a single deployed service then we should change it as a parameter